File: fib.rb
def fib(n)
  return 0 if n == 0
  return 1 if n == 1
  
  sum = fib(n - 1) + fib(n - 2)
end

puts fib(0)
puts fib(1)
puts fib(2)
puts fib(3)
puts fib(4)
puts fib(5)

File: fib.js
function fib(n) {
  if (n <= 1) {
    return 1;
  }
  return fib(n - 1) + fib(n - 2);
}

console.log(fib(0));
console.log(fib(1));
console.log(fib(2));
console.log(fib(3));
console.log(fib(4));